modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotFutureTask.java

```diff
@@ -852,5 +867,28 @@ public boolean touch(long off) {
                     return true;
             }
         }
+
+        /** {@inheritDoc} */
+        @Override public String toString() {
+            StringBuilder sb = new StringBuilder();
+
+            sb.append("AtomicBitSet[arr=[");
+
+            int iMax = arr.length() - 1;
+
+            for (int idx = 0; ; idx++) {
```

Comment:
I think is not a good idea to dump all pages' status. It's resource-consuming and makes logs unreadable in case of large partition sizes. What problems can we solve by having this information?